tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = unit, base, python, node, go, openjdk, openjdk-devel, multistage, fetch-all, list-all
+envlist = unit, base, python, node, go, openjdk, openjdk-devel, multistage, size, fetch-all, list-all
```

Comment:
we'll need to change tox. Either its own env to not fetch language containers matching the name 'size'. Alternatively, we can probably change the default testenv (so that it doesn't run commands). If we rename the language container tests, we might have something simpler.

Reply:
Yeah, we need to change something here, because at the moment the tests fail as the images are not present. Would it be feasible to run fetch-all beforehands? Eventually we'd want to test all containers anyway, right?